doc/man3/X509_VERIFY_PARAM_set_flags.pod

```diff
@@ -145,12 +145,16 @@ To obtain all names start with B<n> = 0 and increment B<n> as long as no NULL
 pointer is returned.
 
 X509_VERIFY_PARAM_set1_host() sets the expected DNS hostname to
-B<name> clearing any previously specified hostname.  If
-B<name> is NULL, or empty the list of hostnames is cleared, and
+B<name>, clearing any previously specified hostname.
+If B<name> is NULL or empty, the list of hostnames is cleared, and
```

Comment:
Isn't NULL here equal to empty?

Reply:
At this point in the text I just moved a comma and re-formatted the lines, without semantic changes.
Since `name` is a string pointer, in this context "empty" means a string of zero length (`""`),
which is different from the NULL pointer, but for this function has the same effect.

Reply:
To make crystal clear what is meant by 'empty', I've meanwhile extended it to 'the empty string', 
as was already done in `SSL_set1_host.pod`.

BTW, I just had a look at the implementation of `int_x509_param_set_hosts()`,
which does some strange extra checks on the string input including its (optional) length parameter.